.travis.yml

```diff
@@ -1,10 +1,12 @@
 language: objective-c
-osx_image: xcode7.3
+osx_image: xcode8.2
 install:
 #- brew install python # Looks like python is already installed now.
-- brew install python3
-- pip install nose
-- pip install tox
+- pip install --user nose
+- pip install --user tox
+- brew install --user python3
+- pip3 install --user nose
```

Comment:
By not updating and installing `tox/nose` using `pip3` I kept getting this error on travis: 
```
ERROR:   py35: InterpreterNotFound: python3.5
```

I don't have too much experience with python but as far as I can tell, if we want these packages to be available for both 2.7 and 3.6, we need to install it twice.